mlflow/utils/autologging_utils.py

```diff
@@ -7,6 +7,11 @@
 
 
 INPUT_EXAMPLE_SAMPLE_ROWS = 5
+FAILED_INPUT_EXAMPLE_PREFIX_TEXT = "Failed to gather input example: "
+FAILED_MODEL_SIGNATURE_PREFIX_TEXT = "Failed to infer model signature: "
+ENSURE_AUTOLOGGING_ENABLED_TEXT = (
```

Comment:
Nit: This is unused. Did we intend to swap it into `xgboost.py` and `lightgbm.py`?